packages/core/src/tools/create-lambda-handler.js

```diff
@@ -30,17 +30,17 @@ const isRequestOrFunction = (obj) => {
 
 const extendAppRaw = (base, extension) => {
   const keysToOverride = [
-    'test',
+    'inputFields',
+    'outputFields',
     'perform',
     'performList',
     'performSubscribe',
     'performUnsubscribe',
+    'sample',
+    'test',
+    'throttle',
```

Comment:
`inputFields`, `outputFields`, `sample`, and `throttle` don't seem to do anything in this array. In the later `if`, we have this condition:

```
if (
  keysToOverride.indexOf(key) !== -1 &&
  _.isPlainObject(srcValue) &&
  _.isPlainObject(objValue) &&
  isRequestOrFunction(srcValue) &&
  isRequestOrFunction(objValue)
)
```

- `inputFields` and `outputFields` would pass `keysToOverride.indexOf(key) !== -1`, but they won't pass `_.isPlainObject()`.
- `sample` and `throttle` won't pass `isRequestOrFunction()`.

I verified it with:

```js
const base = {
  triggers: {
    recipe: {
      operation: {
        sample: {
          id: 1,
        },
      },
    },
  },
};

const extension = {
  triggers: {
    recipe: {
      operation: {
        sample: {
          name: 'Pancakes',
        },
      },
    },
  },
};

const result = extendAppRaw(base, extension);
console.log(JSON.stringify(result, null, 2));
```

The output is:

```json
{
  "triggers": {
    "recipe": {
      "operation": {
        "sample": {
          "id": 1,
          "name": "Pancakes"
        }
      }
    }
  }
}
```

which is not what we want -- we want to fully overwrite the `operation.sample` object.

Also consider the case where a trigger/create/search key is `sample` too:

```js
const base = {
  triggers: {
    sample: {
      operation: {
        sample: {
          id: 1,
        },
      },
    },
  },
};

const extension = {
  triggers: {
    sample: {
      display: {
        label: 'Sample Trigger',
      },
      operation: {
        sample: {
          name: "sample",
        },
      },
    },
  },
};
```

The expected output is:

```json
{
  "triggers": {
    "sample": {
      "display": {
        "label": "Sample Trigger"
      },
      "operation": {
        "sample": {
          "name": "sample"
        }
      }
    }
  }
}
```

I'm not sure if `_.mergeWith()` can do that. If not, we might have to write our own deep merge function.

Reply:
Thanks for pointing that out @eliangcs ! I had made a note that `inputFields` wouldn't get past the first `if` statement and forgot to come back to it 🤦‍♀️